### PR TITLE
Include AdSense for Platforms meta tag.

### PR DESCRIPTION
**Pull request by kuasha420**
## Summary

Addresses issue #3688

## Checklist

- [ ] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [x] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

File: includes/Modules/AdSense.php [hunk 1]
```diff
@@ -862,4 +864,17 @@ public static function normalize_client_id( $account_id, $client_id ) {
 		return 'accounts/' . $account_id . '/adclients/' . $client_id;
 	}
 
+	/**
+	 * Outputs the Adsense for Platforms meta tags.
+	 *
+	 * @since n.e.x.t
+	 */
+	private function render_amp_platform_meta_tags() {
```
Comment (eugene-manuilov) [inline includes/Modules/AdSense.php:872]: @kuasha420, IB says that it should be named as `render_platform_meta_tags`. Is there a reason why this has been called "render_**amp**_platform_meta_tags"?

**Review by eugene-manuilov — CHANGES_REQUESTED** [review]
Thanks, @kuasha420. I see that this PR doesn't include tests defined in IB:

```
Add tests for checking existence of the meta tags when adsense module is activated (whether fully connected or not).
```

You can capture head output by using the `capture_action` helper function: https://github.com/google/site-kit-wp/blob/465534280dcc889728c4380da0232e43cdc047fa/tests/phpunit/includes/TestCase.php#L149-L164

Just call it with `wp_head` action as we do it in some tests for AdSense: https://github.com/google/site-kit-wp/blob/465534280dcc889728c4380da0232e43cdc047fa/tests/phpunit/integration/Modules/AdSenseTest.php#L161

File: includes/Modules/AdSense.php (hunk 1 above)
Comment (kuasha420) [reply includes/Modules/AdSense.php:872]: That's a mistake as far as I can tell. Will fix it in the next commit.

File: tests/phpunit/integration/Modules/AdSenseTest.php [hunk 2]
```diff
@@ -171,6 +171,58 @@ public function test_block_on_consent_non_amp( $enabled ) {
 		}
 	}
 
+	/**
+	 * @dataProvider module_is_connected
+	 * @param bool $connected
+	 */
+	public function test_adsense_platform_tags( $connected ) {
```
Comment (eugene-manuilov) [inline tests/phpunit/integration/Modules/AdSenseTest.php:178]: @kuasha420 connection status doesn't matter for platform tags. Let's get rid of it.

**Review by eugene-manuilov — CHANGES_REQUESTED** [review]
Thanks, @kuasha420. This is almost good to go. The last request before we can merge it.

File: tests/phpunit/integration/Modules/AdSenseTest.php (hunk 2 above)
Comment (kuasha420) [reply tests/phpunit/integration/Modules/AdSenseTest.php]: Done. :)

**Review by eugene-manuilov — APPROVED** [review]
LGTM. Thanks, @kuasha420.

**Pull request merged**